worker/httpserver/manifold.go

```diff
@@ -99,11 +111,27 @@ func (config ManifoldConfig) start(context dependency.Context) (_ worker.Worker,
 		return nil, errors.Trace(err)
 	}
 
-	var clock clock.Clock
-	if err := context.Get(config.ClockName, &clock); err != nil {
+	var mux *apiserverhttp.Mux
+	if err := context.Get(config.MuxName, &mux); err != nil {
 		return nil, errors.Trace(err)
 	}
 
+	// We don't actually need anything from these workers, but we
+	// shouldn't start until they're available.
+	if err := context.Get(config.APIServerName, nil); err != nil {
+		return nil, errors.Trace(err)
+	}
+	// Only check for the raft transport if raft is enabled.
+	var raftEnabled engine.Flag
+	if err := context.Get(config.RaftEnabledName, &raftEnabled); err != nil {
+		return nil, errors.Trace(err)
+	}
+	if raftEnabled.Check() {
+		if err := context.Get(config.RaftTransportName, nil); err != nil {
+			return nil, errors.Trace(err)
+		}
+	}
+
 	var stTracker workerstate.StateTracker
```

Comment:
I don't think we need the state tracker any more. I think it was juts there to create the authenticator.

Reply:
I was hoping to get rid of it too, but unfortunately we still need it for `newTLSConfig`.